Gemfile

```diff
@@ -30,17 +30,17 @@ gem 'ancestry', '2.0.0'
 gem 'gds-api-adapters', '20.1.1'
 gem 'statsd-ruby', '1.1.0'
 gem 'unicorn', '4.9.0'
-gem 'sidekiq', '2.17.2'
-gem 'sidekiq-statsd', '0.1.2'
+gem 'sidekiq', '~> 3.5.0'
+gem 'sidekiq-statsd', '0.1.4'
 gem 'activejob-retry'
 
-gem 'redis', '3.0.6'
+gem 'redis', '~> 3.2.1'
```

Comment:
Looking at the changelog for redis, they don't appear to be following semver: https://github.com/redis/redis-rb/blob/master/CHANGELOG.md